projects/exui-common-lib/src/lib/models/address.model.ts

```diff
@@ -0,0 +1,10 @@
+// tslint:disable:variable-name
+export class AddressModel {
+  public addressLine1 = '';
```

Comment:
The class should have members defined and not initialised?

Reply:
this is consistent with the toolkit component but I will change